src/main/java/uk/gov/hmcts/reform/prl/services/BulkPrintService.java

```diff
@@ -48,13 +48,7 @@ public UUID send(String caseId, String userToken, String letterType, List<Docume
 
         String s2sToken = authTokenGenerator.generate();
         final List<String> stringifiedDocuments = documents.stream()
-            .map(docInfo -> {
-                try {
-                    return getDocumentsAsBytes(docInfo.getDocumentBinaryUrl(), userToken, s2sToken);
-                } catch (IOException e) {
-                    throw new RuntimeException(e);
-                }
-            })
+            .map(docInfo -> getDocumentsAsBytes(docInfo.getDocumentBinaryUrl(), userToken, s2sToken))
```

Comment:
you have missed the return keyword. double check the keyword, and test the code. verify with @shashisk97 once

Reply:
As we discussed in the standup, return is not need.